examples/ex01_inputfile/src/main.C

```diff
@@ -45,6 +45,6 @@ int main(int argc, char *argv[])
 
   // Free up the memory we created earlier
   delete app;
-
+  // test only without reason
```

Comment:
I don't think you intended to change ex01 on this branch, so please drop this change.